README.md

```diff
@@ -40,97 +38,102 @@ Do you want to install Android's dependencies? [y|N]
 ```
 
 ## What it sets up
+
 ### Default
-* [Ruby] stable for writing general-purpose code
```

Comment:
I kept the formatting changes done automatically by VSCode, happy to revert it if not wanted!

Reply:
This change is fine. Next time, use the section `Insights` to explain the nature of the changes.